tasks.py

```diff
@@ -691,7 +691,7 @@ def lint(ctx, env="container"):
     convenient to install the golangci-lint binaries on the host. This can be
     achieved by running `inv lint --env host`.
     """
-    version = "1.50.1"
+    version = "1.51.2"
```

Comment:
could you run 1.52.1? I've seen some additional restrictions in revive with this version

Reply:
sure

Reply:
done. it didn't require me to make any changes.